### DISCUSSION TITLE
Do environment wait timers increase costs for private repos?

### DISCUSSION THREAD

**Discussion by kevcao-certik** [community/community#36711] [Actions]
If I create an environment with a wait timer and then trigger a job running that environment in a workflow, then according to the docs, it appears it does count toward the workflow run time:

> **Workflow run time** - Each workflow run is limited to 35 days. If a workflow run reaches this limit, the workflow run is cancelled. This period includes execution duration, and time spent on waiting and approval.

I just wanted to double check if this means that for private repos, if I would be billed for that wait time then.

**Comment by ossanna16** [comment]
Hey there @kevcao-certik and thank you for asking a good question! I'm glad you found our community 🙂
